saleor/graphql/product/types.py

```diff
@@ -99,6 +100,17 @@ def resolve_attribute_value_type(attribute_value):
     return AttributeValueType.STRING
 
 
+def resolve_background_image(background_image, alt, size, info):
+    if background_image:
```

Comment:
I'd move that check outside of this function and make it always return an `Image`.